### DISCUSSION TITLE
Docs: enable Svelte and Stencil recipes in the sidebar

### DISCUSSION THREAD

**Discussion by fvsch** [statelyai/xstate#1612] [Ideas] [answered]
Currently the “Recipes” section of the docs has 4 pages:

![Screenshot 2020-11-02 at 18 37 17](https://user-images.githubusercontent.com/243601/97900833-8b2a1900-1d3b-11eb-9eab-662abf75b3ff.png)

But the repository has recipes for Svelte and Stencil as well:
https://github.com/davidkpiano/xstate/tree/master/docs/recipes

They’re just not listed in:
https://github.com/davidkpiano/xstate/blob/master/docs/.vuepress/config.js#L84-L89

Should they just be added?

(I spent a couple hours trying to figure out integration with Svelte, and the recipe answers or prevents both issues I’ve had, but I couldn’t find it on the site.)

**Comment by fvsch** [comment]
The pages are published, they're just not in the sidebar:
https://xstate.js.org/docs/recipes/stencil.html
https://xstate.js.org/docs/recipes/svelte.html

**Comment by davidkpiano** [comment, marked as answer]
They should be added now.

**Reply by fvsch** [reply]
Excellent, thanks!
